src/lib/components/ProjectForm/ProjectForm.tsx

```diff
@@ -155,9 +160,13 @@ export const ProjectForm = ({ id }: IProjectFormProps) => {
       const collectionRef = collection(firestore, 'projects');
       const res = await addDoc(collectionRef, preparedData);
       toast.success(DISPLAY_TEXTS.he.toasts[EToastType.AddingNewDoc]);
+      console.log(res);
       router.push({
         pathname: ERoutesNames.Project,
-        query: { [PROJECT_ID_QUERY]: res.id },
+        query: {
+          [PROJECT_ID_QUERY]: res.id,
+          [PROJECT_TYPE_QUERY]: watch(PROJECT_TYPE_QUERY),
```

Comment:
why not `watch(EProjectFields.ProjectType)` ?

Reply:
changed it!